### PR TITLE
Added padding to the Projects

### LINKED ISSUE #295: Improvements required in the fossasia projects sections
**Obvious improvements that are required**

* There is no margin b/w individual projects that should be 20 - 25px
* chat.susi.ai's screenshot has been taken is  dark mode while all other projects are having light backgrounds. chat.susi.ai's screenshot should be in light mode unless mentors want otherwise.

**Would you like to work on the issue?**

I would but anyone else can also work on it

### PR DESCRIPTION
**Pull request by Sid911**
Fixed the squeezing of the fossasia projects

Added padding to the projects by adding`"padding-bottom: 25px"` to css 
fixes #295 

Preview Link : https://sid911.github.io/gci18/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -277,7 +277,7 @@ <h3>Fossasia Attracting Students</h3>
                 <h3> FOSSASIA | Projects </h3>
             </div>
         </div>
-        <div class="row v-align-children">
+        <div class="row v-align-children"style="padding-bottom: 25px">
```
Comment (jogendra) [inline index.html]: Do not hardcode style element in html, rather give it a new class and make relevant changes in CSS file for the defined class.

File: index.html [hunk 2]
```diff
@@ -294,19 +294,19 @@ <h3>SUSI.AI</h3>
                 <a class="btn-filled btn" href="https://chat.susi.ai/">Try</a>
             </div>
         </div>
-        <div class="row v-align-children">
+        <div class="row v-align-children" style="padding-bottom: 25px">
```
Comment (jogendra) [inline index.html]: same here

File: index.html [hunk 3]
```diff
@@ -294,19 +294,19 @@ <h3>SUSI.AI</h3>
...
+            <div class="col-md-4 col-md-offset-1 col-sm-5 col-sm-offset-1" >
                 <h3>Meilix</h3>
                 <p>Meilix is a beautiful and customizable Linux build for out of the box features for an Internet Kiosk. You can use the Meilix Generator (Web app) to make a Linux for your own brand/event, also add apps and features you need pre-installed, it will create an ISO Image of your Linux, which you can use as a live boot or install on PCs.
                 </p>
                 <a class="btn-filled btn" href="https://github.com/fossasia/meilix">Contribute</a>
                 <a class="btn-filled btn" href="https://meilix.fossasia.org/">Try</a>
             </div>
         </div>
-        <div class="row v-align-children">
+        <div class="row v-align-children" style="padding-bottom: 25px">
```
Comment (jogendra) [inline index.html]: here too

File: index.html [hunk 4]
```diff
@@ -319,11 +319,11 @@ <h3>Loklak</h3>
             </div>
             </div>
       </br>
-        <div class="row v-align-children">
+        <div class="row v-align-children" style="padding-bottom: 25px">
```
Comment (jogendra) [inline index.html]: same goes for this

File: index.html [hunk 5]
```diff
@@ -277,7 +277,7 @@ <h3>Fossasia Attracting Students</h3>
                 <h3> FOSSASIA | Projects </h3>
             </div>
         </div>
-        <div class="row v-align-children">
+        <div class="row v-align-children .extra_padding_b">
```
Comment (jogendra) [inline index.html]: you do not need to put `.` before class name here.

**Comment by Sid911** [comment]
@jogendra  removed the . 😊

**Comment by jogendra** [comment]
@Sid911 Provide the preview link.

**Comment by Sid911** [comment]
@jogendra  Added

File: index.html [hunk 6]
```diff
@@ -319,11 +319,11 @@ <h3>Loklak</h3>
             </div>
             </div>
       </br>
-        <div class="row v-align-children">
+        <div class="row v-align-children extra_padding_b" >
             <div class="col-md-7 col-sm-6 text-center mb-xs-24">
                 <img class="cast-shadow" alt="Screenshot" src="img/susper.png" />
             </div>
-            <div class="col-md-4 col-md-offset-1 col-sm-5 col-sm-offset-1">
+            <div class="col-md-4 col-md-offset-1 col-sm-5 col-sm-offset-1" >
```
Comment (fragm3) [inline index.html]: Undo this, avoid adding extra spaces.

**Comment by Sid911** [comment]
@fragm3 Done sir... 😊

**Comment by fragm3** [comment]
@Sid911 The extra space is still there, don't manually resolve it, remove the extra space

**Comment by AakashMallik** [comment]
@Sid911 Material design specs suggest using padding/margin in multiple of 8 or 4. And can't we use margin here, instead of padding? Space should be created by the element, not inside the element. Just a suggestion.

**Comment by fragm3** [comment]
@AakashMallik we are not following material design specs. So any numeric value will work,(better if multiple of 8 or 4). Yes it should be margin. @Sid911

File: css/theme.css [hunk 7]
```diff
@@ -74,6 +74,9 @@ ul.bullets {
 .main-container {
   clear: both;
 }
+.extra_padding_b{
+  padding-bottom: 30px
```
Comment (sakshee-19) [inline css/theme.css]: missing semicolon.

**Comment by Sid911** [comment]
@sakshee-19  @fragm3 Done Sir....

**Comment by prateekiiest** [comment]
Some conflicts still existing. Please resolve them first

**Comment by Sid911** [comment]
@prateekiiest  resolved...

**Comment by jogendra** [comment]
@Sid911 They are still there. Try rebasing too.

**Comment by Sid911** [comment]
@jogendra what are the problems... I can't see a conflict

**Pull request merged**